src/main/java/alfio/controller/api/admin/CheckInApiController.java

```diff
@@ -145,6 +155,7 @@ public boolean revertCheckIn(@PathVariable("eventId") int eventId,
     public ResponseEntity<Boolean> revertCheckIn(@PathVariable("eventName") String eventName,
                                  @PathVariable("ticketIdentifier") String ticketIdentifier,
                                  Principal principal) {
+        accessService.checkEventTicketIdentifierMembership(principal, eventName, ticketIdentifier, AccessService.CHECKIN_ROLES);
```

Comment:
checkEventTicketIdentifierMembership could return an EventAndOrganizationId, so that we can reduce the number of queries